testsuite/replay-verify/main.py

```diff
@@ -234,6 +234,8 @@ def start(self) -> None:
             f"{self.config.timeout_secs}",
             "--block-cache-size",
             "10737418240",
+            "--config-file",
+            "replay_verification_config.json",
```

Comment:
```suggestion
            "/usr/local/bin/replay_verification_config.json",
```
I would suggest moving the JSON file elsewhere, actually.

Reply:
sgtm